test_utils_helpers.py

```
@@ -0,0 +1,1451 @@
+        assert len(hash_result) == 64
+        
+        # Performance should be reasonable (adjust thresholds as needed)
+        assert json_time < 10.0, f"JSON serialization too slow: {json_time}s"
+        assert flatten_time < 10.0, f"Flattening too slow: {flatten_time}s"
+        assert hash_time < 5.0, f"Hashing too slow: {hash_time}s"
+
+
+# Add marker for slow tests
+pytest.mark.slow = pytest.mark.skipif(
+    not pytest.config.getoption("--run-slow", default=False),
+    reason="Slow tests skipped unless --run-slow option provided"
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)